### PR TITLE
Update list of core committers

### PR DESCRIPTION
**Pull request by saturninoabril**
Update list of core committers and their dev areas

### CODE REVIEW

File: source/developer/core-developer-handbook.md [hunk 1]
```diff
@@ -39,6 +39,15 @@ Below is the list of core developers working on Mattermost:
 - **Jonathan Fritz**
   - @jonathan on [pre-release.mattermost.com](https://pre-release.mattermost.com/core/messages/@jonathan) and [@MusikPolice](https://github.com/MusikPolice) on GitHub
   - Dev areas: Desktop
+- **Martin Kraft**
+  - @martin.kraft on [pre-release.mattermost.com](https://pre-release.mattermost.com/core/messages/@martin.kraft) and [@mkraft](https://github.com/mkraft) on GitHub
+  - Dev areas: All areas
```
Comment (jasonblais) [inline source/developer/core-developer-handbook.md]: @saturninoabril Is there any specific area Martin is concentrating on? Like perf measuring / RN apps / permissions?

Comment (mkraft) [reply source/developer/core-developer-handbook.md]: I'm on the permissions team. I have thus far done work in everything except desktop.

Comment (saturninoabril) [reply source/developer/core-developer-handbook.md]: @jasonblais Not sure about it. I just collected individual inputs on dev areas.

Comment (jasonblais) [reply source/developer/core-developer-handbook.md]: @mkraft Any specific area you're concentrating on?

The dev areas will be used by PMs to assign PR reviewers, among other things.

Comment (mkraft) [reply source/developer/core-developer-handbook.md]: @jasonblais No. You can leave it as-is, blank, or choose some areas that no one else wants.

**Pull request merged**